requirements/tests.in

```diff
@@ -1,7 +1,9 @@
 pip-tools
 pip-sync-faster
-coverage
+pytest-coverage
```

Comment:
Like we did in LMS, using `pytest-coverage` instead of using `coverage` directly because `pytest-coverage` knows how to combine the coverage from `pytest-xdist`'s parallel workers